### DISCUSSION TITLE
EL1 Feedback

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#217] [Show and tell]
https://teambasedinquirylearning.github.io/precalculus/2024/EL1.html

Author: @kathypinzon 
Reviewer: @tdegeorge

**Comment by tdegeorge** [comment]
Remark 5.1.1. - there is an extra "and" in the second sentence.

**Comment by tdegeorge** [comment]
Activity 5.1.2 - you have an extra zero in answers (parts c, d, and e) for Job 1. It should be $10,000, not $10,0000

Do we also want to include answers for parts a and b?

**Comment by tdegeorge** [comment]
Remark 5.1.3 - should it say "This idea of "**a**" function that grows very rapidly by a factor, ratio, or percent each time.."? or "This idea of function**"s"** **growing** very rapidly by a factor, ratio, or percent each time...?

**Comment by tdegeorge** [comment]
Activity 5.1.7 - should "table" be "tables" in part c?

**Comment by tdegeorge** [comment]
Activity 5.1.11 - extra "to" in the first sentence.

Parts d and e ask the same question. Did you mean to delete one of them?

Part f - should we put an aside or something to help students recall what a ratio is?

**Comment by tdegeorge** [comment]
Activity 5.1.13 - this activity seems to jump in asking them to find the equation when given two points. Can we make it more scaffolded? Like one part asking what the initial value is to help them create the base equation and then another part guiding them to find the b-value?

**Comment by tdegeorge** [comment]
Definition 5.1.15 - one of the "e"s is not in math mode.

**Comment by tdegeorge** [comment]
Activity 5.1.16 - could we make this activity MC? I feel like students could compute these in a few different ways.

**Comment by tdegeorge** [comment]
Activity 5.1.18 - Part b should say to fill in the table for "g(x)", not "f(x)"
